<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissionWidget</class>
 <widget class="QWidget" name="MissionWidget">
  <property name="minimumSize">
   <size><width>0</width><height>150</height></size>
  </property>
  <property name="maximumWidth">
   <number>800</number>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed"/>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">

   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDroneType">
       <property name="text"><string>Drone: quad-fpv</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelLocation">
       <property name="text"><string>Location: Asuncion</string></property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelMissionGroup">
       <property name="text"><string>Group: default</string></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelStatus">
       <property name="text"><string>Status: in progress</string></property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="labelTimeWindow">
       <property name="text"><string>Time Window: 14:00-15:00 UTC</string></property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="labelMissionId">
       <property name="text"><string>ID: 4505bfee-xxxx-xxxx</string></property>
      </widget>
     </item>
    </layout>
   </item>

   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="spacerLeft">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint">
        <size><width>40</width><height>20</height></size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonConnect">
       <property name="text"><string>Connect</string></property>
      </widget>
     </item>
     <item>
      <spacer name="spacerRight">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint">
        <size><width>40</width><height>20</height></size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
